src/ui/digital-exchange/common/StarRating.js

```diff
@@ -0,0 +1,25 @@
+import { range } from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import StarIcon from 'ui/digital-exchange/common/StarIcon';
+
+const StarRating = ({ rating }) => (
+  <div>
```

Comment:
```suggestion
  const roundedRating = Math.round(rating)
  <div aria-label={roundedRating}>
```

(please double-check this suggestion)